arangod/CMakeLists.txt

```diff
@@ -283,6 +283,7 @@ set(LIB_ARANGO_PREGEL_SOURCES
   Pregel/Algos/SSSP.cpp
   Pregel/Algos/ShortestPath.cpp
   Pregel/Algos/WCC.cpp
+  Pregel/Algos/ReadWrite.cpp
```

Comment:
if we exclude this from released builds this needs to be moved into an `if (USE_MAINTAINER_MODE) ` as welll